samples/metrics_monitor/CMakeLists.txt

```diff
@@ -28,30 +28,16 @@ pkg_check_modules(PKG_LIBDRM    libdrm)
 pkg_check_modules(PKG_PCIACCESS pciaccess)
 
 if( PKG_LIBDRM_FOUND AND
-    PKG_PCIACCESS_FOUND AND
-    DEFINED GTEST_FOUND)
+    PKG_PCIACCESS_FOUND )
 
-    set( sources "" )
+    file( GLOB_RECURSE test_srcs "${CMAKE_CURRENT_SOURCE_DIR}/test/*.cpp" )
 
-    list ( APPEND incldirs
-        ${GTEST_INCLUDE}/gtest
-        ${GTEST_INCLUDE}/gtest/internal
-        ${CMAKE_CURRENT_SOURCE_DIR}/test )
+    add_executable( test_monitor ${test_srcs})
+    target_include_directories( test_monitor PUBLIC ./include ${PKG_LIBDRM_INCLUDE_DIRS})
```

Comment:
Why [PUBLIC]? Are u going to link [test_monitor] to other targets?

Reply:
Done